docs/docs/getting-started.md

```diff
@@ -110,58 +68,12 @@ Then reference it from `helloworld.html`:
 
 Note that some browsers (Chrome, e.g.) will fail to load the file unless it's served via HTTP.
 
-### Offline Transform
-
-First install the [Babel](http://babeljs.io/) command-line tools (requires [npm](https://www.npmjs.com/)):
+## Using React with NPM / Bower
 
-```
-npm install --global babel-cli
-npm install babel-preset-react
-```
-
-Then, translate your `src/helloworld.js` file to plain JavaScript:
-
-```
-babel --presets react src --watch --out-dir build
-```
-
-> Note:
->
-> If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
-
-The file `build/helloworld.js` is autogenerated whenever you make a change. Read the [Babel CLI documentation](http://babeljs.io/docs/usage/cli/) for more advanced usage.
-
-```javascript{2}
-ReactDOM.render(
-  React.createElement('h1', null, 'Hello, world!'),
-  document.getElementById('example')
-);
-```
-
-
-Update your HTML file as below:
-
-```html{8,12}
-<!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8" />
-    <title>Hello React!</title>
-    <script src="build/react.js"></script>
-    <script src="build/react-dom.js"></script>
-    <!-- No need for Babel! -->
-  </head>
-  <body>
-    <div id="example"></div>
-    <script src="build/helloworld.js"></script>
-  </body>
-</html>
-```
+You can also use React with package managers like NPM and/or Bower.  You can learn more in our [Package Managers](/react/docs/package-management.html) section.
```

Comment:
Let's not "and/or" this. Let's pick one ("or" is my preference because I hope you aren't using both together for client-side development).